index.html

```diff
@@ -7,92 +7,207 @@
       </div>
-      <div class="btn"><button>Book now</button></div>
     </div>
     <div class="right_side">
       <img class="mountains_pic" src="mountain_home.png" alt="widok na góry">
     </div>
-  </div>
-  <div class="fourth_page_container">
+  </section>
+
+
+
+
+
+  <section class="section fourth_page_container">
     <div class="left_side_fourth_page">
       <div><img class="girl_pic" src="girl.png" alt="zdjęcie dziewczyny"></div>
-      <div class="newsletter">
-        <label for="newsletter_header"> newsletter comming soon</label>
-        <input type="email" class="email" placeholder="your e-mail address">
-      </div>
+
     </div>
+    
+
+
+
+
+
     <div class="right_side_fourth_page">
-      <h1 class="cabin_lovers">+2,000<br>cabin lovers</h1>
-      <p class="fourth_page_footer">Marina Skovic, Visual Artist</p>
-      <div class="bot_nav_container">
-        <nav class="bot_nav"></nav>
+      <div class="fan_number">
+        <p class="cabin_lovers">+2,000<br>cabin lovers</p>
+      </div>
+      <div class="autor">
+        <footer class="fourth_page_footer">Marina Skovic, Visual Artist</footer>
+        
+        <ul class="unordered_list_for_numbers">
+          <li class="numbers_fourth_page"><a class="anchor_number" href="#">1</li></a>
+          <li class="numbers_fourth_page"><a class="anchor_number" href="#">2</li></a>
+          <li class="numbers_fourth_page"><a class="anchor_number" href="#">3</li></a>
+        </ul>
+      </div></div>
+  </section>
+
+
+
+  <section class="section fifth_page_container">
+    <div class="newsletter">
+      <div class="input_newsletter">
+        <div class="input_header">
+          <label for="newsletter_header" class="newsletter_info"> newsletter comming soon</label>
+        </div>
+        <div class="input_input">
+          <input type="email" class="email" placeholder="your e-mail address">
+        </div>
       </div>
     </div>
-  </div>
 
-</body>
+    <footer class="bot_nav_container">
+      
+        <ul class="ul_container">
+          <li class="nav">Help Center</li>
+          <li class="nav">Facebook</li>
+          <li class="nav">About</li>
+        </ul>
+        <ul class="ul_container">
+          <li class="nav">FAQ</li>
+          <li class="nav">Contact</li>
+          <li class="nav">Instagram</li>
+        </ul>
+        <ul class="ul_container">
+          <li class="nav">Blog</li>
+          <li class="nav">Reviews</li>
+          <li class="nav">Twitter</li>
+    </div>
 
+  </section>
 
 
+
+
+
+
+
+<div class="section">
+  <div class="last_footer_wrapper">
+
+    <div class="last_footer">
+      <div class="footer_box">
+        <ul class="footer_list">
+        <li class="footer_par"><a href="#">&#169 2020 Remotely</li></a>
+        <li class="footer_par"><a href="#">Privacy Policy</li></a>
+        <li class="footer_par"><a href="#">Terms and Conditions</li></a>
+      </ul></div></footer>
+    </div></div>
```

Comment:
1. fix formatting, this is a mess, 
2. remove this opacity and set correct color (still not done ⛔)

Reply:
fixed